### PR TITLE
add spacing for units

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
https://linear.app/electricitymaps/issue/ELE-1557/app-inconsistent-spacing-on-numbers-and-units-throughout-the-app
## Description
Improves the spacing to add consistency of spacing between units, gathered some feedback from sciency and designy people in the office :)

### CODE REVIEW

**Review by madsnedergaard — COMMENTED** [review]
LGTM! There's a minor merge conflict on the Square as I accidentally committed changes in #138 directly to `master` 😬 
Maybe we should merge that one first then and apply your change on top?

File: web/src/utils/formatting.ts [hunk 1]
```diff
@@ -9,7 +9,10 @@ export const formatPower = function (d: number, numberDigits = DEFAULT_NUM_DIGIT
   if (d == undefined || Number.isNaN(d)) {
     return d;
   }
-  return `${d3.format(`.${numberDigits}s`)(d * 1e6)}W`;
+  const power = `${d3.format(`.${numberDigits}s`)(d * 1e6)}W` //Add a space between the number and the unit
```
Comment (madsnedergaard) [inline web/src/utils/formatting.ts:12]: I assume we can't just do `.${} s`, right? It's kinda weird that they are doing it like this in general 🤔

**Review by madsnedergaard — COMMENTED** [review]
One more comment

File: web/src/utils/formatting.ts (hunk 1 above)
Comment (tonypls) [reply web/src/utils/formatting.ts:12]: Hmm I'm not sure exactly what you mean, which is testament to this being generally hard to understand. 

But it seems to work flawlessly 👍

Comment (madsnedergaard) [reply web/src/utils/formatting.ts:12]: I wanted to add the space inside format, but if this works flawlessly then let's stick to it!

**Review by madsnedergaard — APPROVED** [review]

**Pull request merged**